### PR TITLE
Add user stories

### PR DESCRIPTION
**Pull request by ckiteou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,67 @@
...
+MVP
+------
+
+As a space owner
+So I can rent out my space
+I want to be able to list it
+
+As a space owner
+So I can rent all my spaces
+I want to be able to list multiple spaces
```
Comment (dbugsy) [inline README.md]: I think this could be a second iteration thing, rather than initial MVP

File: README.md [hunk 2]
```diff
@@ -16,6 +16,67 @@
...
+I want to be able to login and logout
+
+As a guest
+So I can manage my booking(s)
+I want to be able to login and logout
+
+Extras 2
+------
+
+As a space owner
```
Comment (dbugsy) [inline README.md:72]: This seems essential - the app doesn't work if multiple bookings can happen.

Comment (petrakh) [reply README.md:72]: implemented, thanks! 🐷

**Review by BDCraven — APPROVED** [review]
🐷

**Pull request merged**